px-datetime-picker.html

```diff
@@ -276,6 +346,70 @@
       if(evt.detail.dateOrTime === "Date") {
         this.$.dropdown.toggle();
       }
+    },
+    /**
+     */
+    _collapseQueryIsValid: function(query) {
+      if (typeof query === 'number') {
+        return true;
+      }
+      if (typeof query === 'string' && query.slice(-2) === 'px' && !isNaN(parseInt(query))) {
+        return true;
```

Comment:
Is this dummy-check necessary? We've declared collapseAt as a Number, is it even possible to assign a string to it?